src/styles/blueprint-overrides.css

```diff
@@ -52,4 +52,8 @@
   .bp3-dark .bp3-popover .bp3-popover-arrow-fill {
     fill: var(--primary-9-dark);
   }
+
+  .bp3-dialog {
+    width: auto !important;
```

Comment:
Do we need to use !important?

Reply:
Yes, that's the whole point—we need to override the width using higher specificity.

Reply:
Upon checking at multiple places, `important` was overriding even inline styles.
Changing this to use a CSS trick for a rule with 2 classes to increase specificity.
`.bp3-dialog.bp3-dialog {
    width: auto;
  }`